packages/ui/src/components/data-display/timer/index.tsx

```diff
@@ -53,7 +53,9 @@ export const Timer = ({
                     className={iconStyles({ className: className?.icon })}
                 />
             )}
-            {duration.format("DD[D] HH[H] mm[M] ss[S]")}
+            {duration.asSeconds() >= 0
+                ? duration.format("DD[D] HH[H] mm[M] ss[S]")
+                : "00D 00H 00M 00S"}
```

Comment:
I think we use this format in several places, maybe we should make this a variable and use it everywhere.

something like this:
`const ZERO_DATE_TIME_FORMAT=00D 00H 00M 00S`